nilearn/masking.py

```diff
@@ -691,6 +710,76 @@ def compute_multi_gray_matter_mask(target_imgs, threshold=.5,
     mask: nibabel.Nifti1Image
         The brain mask (3D image)
 
+    See also
+    --------
+    nilearn.masking.compute_brain_mask
+    """
+    return compute_multi_brain_mask(target_imgs=target_imgs,
+                                    threshold=threshold, connected=connected,
+                                    opening=opening, memory=memory,
+                                    verbose=verbose, n_jobs=n_jobs,
+                                    mask='whole-brain', **kwargs)
+
+
+def compute_multi_brain_mask(target_imgs, threshold=.5, connected=True,
```

Comment:
Same comment for this function.

Reply:
Here, I replaced `compute_multi_gray_matter_mask` by the new function, i.e. `compute_multi_brain_mask`.